shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-statement/src/main/java/org/apache/shardingsphere/sharding/distsql/parser/statement/DropShardingAlgorithmStatement.java

```diff
@@ -30,5 +30,12 @@
 @Getter
 public final class DropShardingAlgorithmStatement extends DropRuleStatement {
     
+    private final boolean containsExistsClause;
+    
```

Comment:
Hi, in the latest code `containsExistsClause` has been added in the `DropStatement`